Pipfile

```diff
@@ -9,17 +9,17 @@ docker = "==4.1.0"
 [packages]
 robotframework = "~=5.0.1"
 robotframework-seleniumlibrary = "~=6.0.0"
-requests = "~=2.27.1"
+requests = "==2.28.1"
 python-dotenv = "~=0.20.0"
-robotframework-pabot = "~=2.5.3"
+robotframework-pabot = "==2.6.0"
```

Comment:
We will want to keep the `~=` syntax instead of `==`. This means that we follow semver so that pipenv locks the major version of the package.